skglm/solvers/cd_solver.py

```diff
@@ -347,118 +347,6 @@ def cd_solver(
     return w, np.array(obj_out), stop_crit
 
 
-@njit
-def dist_fix_point(w, grad, datafit, penalty, ws):
-    """Compute the violation of the fixed point iterate scheme.
```

Comment:
also, revert these

Reply:
I 100% agree with you, but let's do it in another PR.
IMO it makes the diff terribly and hard to follow.